isaric/parsers/isaric-ecmo.toml

```diff
@@ -239,13 +239,12 @@
   observation = { kind = "oneToMany", schema = "../../schemas/dev/observation.schema.json", common = { visit_id = { field = "subjid", sensitive = true } } }
 
 [subject]
-  country_iso3 = { field = "country", ref = "countryMap" }
   pathogen = "COVID-19"
 
   [subject.sex_at_birth]
     field = "sex"
     description = "Sex at Birth"
-    values = { 1 = "male", 2 = "female", 3 = "non_binary" }
```

Comment:
Was this (3 = "non_binary") being mapped this way because it was linked to an _enum_ variable?

Reply:
In all 3 of the ISARIC parsers (CCPUK, RAPID and ECMO) 3 was being recorded as non-binary, but looking back at the data dict 3 is "not specified", so this is just correcting the mistake. Having taken those out none of the existing parsers have non-binary as an option for sex, and as discussed in the last meeting non-binary isn't a valid response to a sex *at birth* field, so I removed that from the schema.